### ISSUE TITLE
[Desktop] some NTP widget menus do not look nice with translations

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#10749]
## Description 

Some of the NTP widget menus do not look nice when the translation is long, causing it to go to two lines. Would be nice if these menus could be dynamic enough to handle two lines (or more?) if needed. If you hover over one option, it should be nicely highlighted as it is in English.

## Steps to Reproduce

pre-req: be using a language such as French
   1. Open NTP
   2. Click on the widget menu for Rewards

## Actual result:

Because the translation for the first item on the list goes to two lines, the widget menu looks off when you hover over an option. It's not dynamic enough to handle multiple lines as the English translation is just on one. When you hover over either option the highlighting is off. In the screenshot below, you're hovering over the second option but the first option is partially highlighted, so this doesn't look right.
<img width="329" alt="Capture d’écran 2020-07-15 à 10 30 11 AM" src="https://user-images.githubusercontent.com/28145373/87558200-d46c3500-c686-11ea-98d5-35336c1bbabb.png">

## Expected result:
Menus should handle the translations when they go to more than one line - so if a translation is multiple lines and you hover over one, the highlighting should be as it is in English:
<img width="289" alt="Screen Shot 2020-07-15 at 10 43 45 AM" src="https://user-images.githubusercontent.com/28145373/87559312-23669a00-c688-11ea-8a78-958e766944e4.png">

## Reproduces how often: 

easily

## Brave version (brave://version info)

```
Brave | 1.11.97 Chromium: 84.0.4147.89 (Build officiel) (64 bits)
-- | --
Révision | 19abfe7bcba9318a0b2a6bc6634a67fc834aa592-refs/branch-heads/4147@{#852}
Système d'exploitation | macOS Version 10.14.6 (assemblage 18G3020)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? 1.10.x yes, 1.11.x (RC) yes
- Can you reproduce this issue with the beta channel? unsure
- Can you reproduce this issue with the dev channel? unsure
- Can you reproduce this issue with the nightly channel? unsure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by srirambv** [comment]
Similar to #10742

**Comment by rebron** [comment]
cc: @karenkliu We need a bit of room here for translations in the menus.

**Comment by karenkliu** [comment]
@rebron  The menu should really be engineered with dynamic padding to accommodate different text lengths instead of using fixed sizes. Needs a technical update.

**Comment by LaurenWags** [comment]
Marking as `QA/Blocked` as it is part of a PR requested for uplift

**Comment by LaurenWags** [comment]
Removing `QA/Blocked`, this should be available to test in 1.13.75 and above.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.13.79 Chromium: 85.0.4183.69 (Build officiel) dev (64 bits)
-- | --
Révision | 4554ea1a1171bd8d06951a4b7d9336afe6c59967-refs/branch-heads/4183@{#1426}
Système d'exploitation | macOS Version 10.14.6 (assemblage 18G3020)
```

* Verified STR from the issue for the languages below
**`French`**:
<img width="328" alt="Capture d’écran 2020-08-20 à 4 32 25 PM" src="https://user-images.githubusercontent.com/28145373/90822788-f51d4f80-e302-11ea-81be-82aa60d93edf.png">

**`Norwegian`**:
<img width="307" alt="Skjermbilde 2020-08-20 kl  4 35 11 pm" src="https://user-images.githubusercontent.com/28145373/90822907-23029400-e303-11ea-9a8c-ea62a792e4b4.png">

**`Spanish`**:
<img width="318" alt="Captura de Pantalla 2020-08-20 a la(s) 4 35 50 p  m" src="https://user-images.githubusercontent.com/28145373/90822966-39a8eb00-e303-11ea-94ca-ed030e7b920d.png">

**`English`**:
<img width="306" alt="Screen Shot 2020-08-20 at 4 36 27 PM" src="https://user-images.githubusercontent.com/28145373/90823023-4f1e1500-e303-11ea-83f4-6e74cd6374f6.png">

**`N/A` - just wanted to show something that was fairly long**
<img width="1172" alt="Capture d’écran 2020-08-20 à 4 33 34 PM" src="https://user-images.githubusercontent.com/28145373/90822792-f77fa980-e302-11ea-872e-3fc595b107e2.png">

-------
